lib/rubocop/cop/rails/delegate.rb

```diff
@@ -15,19 +20,29 @@ module Rails
       #   # good
       #   delegate :bar, to: :foo
       #
-      #   # bad
-      #   def foo_bar
-      #     foo.bar
-      #   end
-      #
-      #   # good
-      #   delegate :bar, to: :foo, prefix: true
-      #
       #   # good
       #   private
       #   def bar
       #     foo.bar
       #   end
+      #
+      #   # EnforcePrefix: true
+      #     # bad
```

Comment:
I wouldn't nest this additionally, as it doesn't look well in the generated documentation.